[
    "Match found, the package 'python3' is vulnerable to 'CVE-2021-23336' due to default status. - Agent '' (ID: '018', Version: '').",
<<<<<<< HEAD
=======
    "Match found, the package 'python3' is vulnerable to 'CVE-2022-0391' due to default status. - Agent '' (ID: '018', Version: '').",
>>>>>>> a2da170cb4 (refactor: cleaned some auxiliar libraries)
    "Match found, the package 'python3', is vulnerable to 'CVE-2023-27043'. Current version: '3.9.18-1.el9_3' (less than '0:3.9.18-1.el9_3.1' or equal to ''). - Agent '' (ID: '018', Version: '').",
    "Match found, the package 'python3' is vulnerable to 'CVE-2023-36632' due to default status. - Agent '' (ID: '018', Version: '').",
    "Processing and publish key: CVE-2023-36632",
    "Processing and publish key: CVE-2023-27043",
<<<<<<< HEAD
    "Processing and publish key: CVE-2021-23336",
    "No match due to default status for Package: python3, Version: 3.9.18-1.el9_3 while scanning for Vulnerability: CVE-2022-0391"
]
=======
    "Processing and publish key: CVE-2022-0391",
    "Processing and publish key: CVE-2021-23336"
    ]
>>>>>>> a2da170cb4 (refactor: cleaned some auxiliar libraries)

